js/app.js

```diff
@@ -0,0 +1,171 @@
+const menu = document.querySelector('.mobile-menu');
+const hamburger = document.querySelector('.fa-bars');
+const body = document.querySelector('body');
+const featuredSpeakers = document.querySelector('#featured-speakers');
+// const img = document.querySelector('.image-speaker-1');
+
+const speakers = [
+  {
+    name: 'Asha McMillan',
+    job: 'Founder & CEO',
+    description: 'Asha McMillan is the Founder and CEO of the company.',
+    image: './business-people-at-the-office.webp',
+  },
+  {
+    name: 'Kameron Coleman',
+    job: 'Event Manager',
+    description: 'Kameron is the Event Manager of the compay.',
+    image: './pexels-icsa-1709001.webp',
+  },
+  {
+    name: 'Alima Pate',
+    job: 'Marketing Manager',
+    description: 'Alima is the Marketing Manager of the company.',
+    image: './businesswoman-posing-at-the-office.webp',
+  },
+  {
+    name: 'Katy Mustafa',
+    job: 'Art Design Manager',
+    description: 'Katy is responsible for all the images, logos and colors of the company.',
+    image: './pexels-andrea-piacquadio-3756679.webp',
+  },
+  {
+    name: 'Nicholas Boyer',
+    job: 'Web Developer',
+    description: 'Nicholas is the Web Developer of the company.',
+    image: './pexels-mikhail-nilov-7989081.webp',
+  },
+  {
+    name: 'Hannah Finley',
+    job: 'General Artist Manager',
+    description: 'Hannah works directly with the artist to make sure they have all they need.',
+    image: './pexels-thisisengineering-3861962.webp',
+  },
+];
+
+window.onload = () => {
+  const {
+    name, job, description, image,
+  } = speakers[0];
+  const {
+    name: name2, job: job2, description: description2, image: image2,
+  } = speakers[1];
+  const {
+    name: name3, job: job3, description: description3, image: image3,
+  } = speakers[2];
+  const {
+    name: name4, job: job4, description: description4, image: image4,
+  } = speakers[3];
+  const {
+    name: name5, job: job5, description: description5, image: image5,
+  } = speakers[4];
+  const {
+    name: name6, job: job6, description: description6, image: image6,
+  } = speakers[5];
+
+  featuredSpeakers.innerHTML = `
+  <div class="featured-title">
+    <h2>Featured Speakers</h2>
+    <hr class=speakers-hr>
+  </div>
+  <div class="featured-speakers">
+    <div class="speaker">
+      <div class="speaker-image">
+        <img class="image-speaker" src='${image}'></img>
```

Comment:
- [ ] Don't forget to include the `alt` property to all `image` elements. It gives information when the image link is broken. :+1: